src/app/drive/services/limit.service.test.ts

```diff
@@ -1,26 +1,36 @@
-import { describe, it, expect, vi, Mock } from 'vitest';
+import { describe, it, expect, vi, beforeEach } from 'vitest';
 import { SdkFactory } from '../../core/factory/sdk';
 import limitService from './limit.service';
 
+vi.mock('app/core/components/Sidenav/Sidenav', () => ({
+  HUNDRED_TB: 109951162777600,
+}));
```

Comment:
I think it is better to remove this constant from the component, since it does not make sense to export a constant from a component which makes it difficult or may cause problems for testing